tpl/os/init.go

```diff
@@ -55,6 +55,13 @@ func init() {
 			},
 		)
 
+		ns.AddMethodMapping(ctx.FileStat,
+			[]string{"fileStat"},
```

Comment:
Is it better to rename here?, @bep
In my opinion, `fileStat` will be better than `Stat` because  we cannot get what kind of stat. if `Stat` is used in Hugo template.

Reply:
I think we should remove it. We don't need an alias for this, as `os.Stat` is already short enough.